lib/color.js

```diff
@@ -219,6 +218,68 @@ class ColorXY {
         return kelvinToMireds(kelvin);
     }
 
+    /**
+     * Converts CIE color space to RGB color space
+     * From: https://github.com/usolved/cie-rgb-converter/blob/master/cie_rgb_converter.js
+     * @return {ColorRGB}
+     */
+    toRGB() {
```

Comment:
@blazewicz can you have another look at this, going from XY -> RGB I'm not getting the expected red/green/blues, they all seem off a bit and have negative values too which definitely doesn't seem right.

Reply:
I responded in resolved thread, sorry.

https://github.com/Koenkk/zigbee-herdsman-converters/pull/2441/files#r632667617